src/kiwixapp.cpp

```diff
@@ -413,8 +413,13 @@ void KiwixApp::createActions()
     mpa_actions[FindInPageAction]->setShortcuts({QKeySequence::Find, Qt::Key_F3});
     connect(mpa_actions[FindInPageAction], &QAction::triggered,
             this, [=]() { getTabWidget()->openFindInPageBar(); });
-
-    CREATE_ACTION_ICON_SHORTCUT(ToggleFullscreenAction, "full-screen-enter", gt("set-fullscreen"),  QKeySequence::FullScreen);
+    
+    bool fullScreenKeyUndefined = (QKeySequence(QKeySequence::FullScreen).toString()).isEmpty();
+    if (fullScreenKeyUndefined) {
+        CREATE_ACTION_ICON_SHORTCUT(ToggleFullscreenAction, "full-screen-enter", gt("set-fullscreen"), Qt::Key_F11);
+    } else {
+        CREATE_ACTION_ICON_SHORTCUT(ToggleFullscreenAction, "full-screen-enter", gt("set-fullscreen"), QKeySequence::FullScreen);
+    }
```

Comment:
1. Why don't you use [`QKeySequence::isEmpty()`](https://doc.qt.io/qt-5/qkeysequence.html#isEmpty)?
2. The difference between your two `CREATE_ACTION_ICON_SHORTCUT()` lines is only in the last argument and I'd rather have the duplication eliminated by isolating the said difference into a constant that would be passed as the last argument:

```c++
    const auto fullScreenKeySeq = QKeySequence(QKeySequence::FullScreen).isEmpty()
                                ? Qt::Key_F11
                                : QKeySequence::FullScreen;
```

Reply:
It "Looks Good To Me But Could Be Made Simpler By Moving Said Behavior" to me as well! 😃

Maybe it is not a good idea to do PRs late at night after work 😢

Totally agree with you.

Reply:
> It "Looks Good To Me But Could Be Made Simpler By ... Moving Said Behavior" to me as well!

Almost there! Actually, I meant ""Looks Good To Me But Can Be Made Slightly Better"

Reply:
Ah... I thought it would have to be continued below :| Not for me ig 

LGTMNLMKITM 🫡

Reply:
> LGTMNLMKITM 🫡

I can debreviate it in one of three ways but I am not sure which one it actually stands for: 

1. Looks good to me now. Let me know if that's marvelous.
2. Lest `git` triggers mysterious nonsense, let my kittens ingest their milk.
3. Little Grandpa took my neighbour's last mathematical kidney in the meantime.